src/test/kotlin/day03/HeroTest.kt

```
@@ -0,0 +1,29 @@
+package day03
+
+import org.junit.Assert.*
+ class HeroTest {
+
+@org.junit.Before
+ fun setUp() {}
```

detekt (1.23.7)

[warning] This empty block of code can be removed.

(detekt.empty-blocks.EmptyFunctionBlock)